lib/usd/hdMaya/adapters/lightAdapter.h

```diff
@@ -26,7 +26,7 @@
 #include <maya/MFnLight.h>
 #include <maya/MFnNonExtendedLight.h>
 
-#include "dagAdapter.h"
+#include <hdMaya/adapters/dagAdapter.h>
```

Comment:
I think this one would go in its own group *below* `<pxr/imaging/hd/light.h>`?

Reply:
Thanks you @mattyjams . Please see 7cb7810a